Bump version, set release names (#1319)
nf-core/sarek: CHANGELOG.md

The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).

> ## [3.4](https://github.com/nf-core/sarek/releases/tag/3.4) - Pårtetjåkkå

Reviewer: ```suggestion
## [3.4](https://github.com/nf-core/sarek/releases/tag/3.4) - Pårtetjåkko
```

> ## [3.4](https://github.com/nf-core/sarek/releases/tag/3.4) - Pårtetjåkko

Reviewer: `3.4` or `3.4.0`?

Author: we  have been going back and forth on this. 3.0, 3.1, 3.2.0, 3.30 happy to be consistent there.

Reviewer: Let's try to follow the trend we established since `3.2.0`.
But I agree, for me `3.4.0` is the same as `3.4`

> Pårtetjåkkå is mountain in the south of the park.

Reviewer: ```suggestion
Pårtetjåkko is mountain in the south of the park.
```

> Pårtetjåkkå is mountain in the south of the park.

Author: ```suggestion
Pårtetjåkkå is a mountain in the south of the park.
```

> Pårtetjåkkå is a mountain in the south of the park.

Reviewer: ```suggestion
Pårtetjåkko is a mountain in the south of the park.
```

> Pårtetjåkkå is a mountain in the south of the park.

Author: ```suggestion
Pårtetjåkko is a mountain in the south of the park and the Lule Sámi spelling of Bårddetjåhkkå.
```

| `gatk4spark/baserecalibrator` | `GATK4_BASERECALIBRATORSPARK` | `GATK4SPARK_BASERECALIBRATOR` |
| `gatk4spark/markduplicates`   | `GATK4_MARKDUPLICATESSPARK`   | `GATK4SPARK_MARKDUPLICATES`   |

## [3.3.2](https://github.com/nf-core/sarek/releases/tag/3.3.2) - Ráhpaädno

Ráhpaädno is the Lule Sámi spelling of Rapaälven.

> ### Added

Reviewer: why the change?

Author: bad copy paste on my part
